src/main/flight/pid_mw23.c

```diff
@@ -146,10 +149,14 @@ void pidMultiWii23(const pidProfile_t *pidProfile, const controlRateConfig_t *co
         // Delta from measurement
         delta = -(gyroError - lastErrorForDelta[axis]);
         lastErrorForDelta[axis] = gyroError;
-        if (pidProfile->dterm_cut_hz) {
-            // Dterm delta low pass
-            DTerm = delta;
-            DTerm = lrintf(applyBiQuadFilter((float) DTerm, &deltaFilterState[axis])) * 3;  // Keep same scaling as unfiltered DTerm
+        if (pidProfile->dterm_lpf_hz) {
+            // Dterm low pass filter
+            DTerm = delta * 3; // Keep same scaling as unfiltered DTerm
+#ifdef USE_PID_BIQUAD_FILTER
```

Comment:
IMO biquad for dterm can fully be removed. It is too much and it easily puts D and P out of phase. pt1 is a 1 point filter which is good enough in case when filter wanted